### ISSUE TITLE
Multilevel Mixture for Binary dependent variable

### ISSUE THREAD

**Issue by PaoBe** [CecileProust-Lima/lcmm#6]
I need to estimate a multilevel mixture model (or latent class mixed model) and my dependent variable is binary distributed. 
The conditional part of the model, more or less, should be in the form of logit(Y_ijc) = \alpha_c + \beta_c X_ijc + u_jc ,
whereas the overall model is as usuale something like p(y|x)=sum \pi_c * p_c(y|x,\theta_c) where \pi_c are the mixture weights and \theta_c is the vector of the parameters.

Basically, I'm expecting one \alpha and one \beta for each latent group as well as a vector of random intercepts u_i.

I'm not able to find the way to build a similar model using lcmm package.
Thanks for your time.

**Comment by CecileProust-Lima** [comment]
Lcmm does not include the logistic latent class mixed model. Instead, it includes a probit latent class mixed model. It should work equally fine for your case. 
Probit model can be done with lcmm by specifying link = "thresholds". An example of threshold specification is given in the JSS paper ([https://www.jstatsoft.org/article/view/v078i02](https://www.jstatsoft.org/article/view/v078i02)) although this is in the case of an homogeneous model (G=1) and this is for a more general ordinal outcome. 
Cécile

**Comment by PaoBe** [comment]
Thanks for your help.

Paolo
